Directwerk/directwerk-podcast/README.md

```
@@ -132,20 +132,21 @@ tracked before redirecting to the public CDN. Analytics is fail-open and never g
 
 ---
 
-## Feed builder (deferred)
+## Feed builder (Formate)
 
-Subscribers will compose a **private** RSS feed by selecting formats (and optionally categories).
-That feed only includes episodes they are **entitled** to and that match the filter.
+Subscribers compose a **private** RSS feed by selecting one or more **Formate**. That feed only
+includes episodes they are **entitled** to and that match at least one selected active format.
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)